src/main/java/me/dinowernli/grpc/polyglot/config/CommandLineArgs.java

```diff
@@ -214,4 +251,17 @@ public boolean isHelp() {
     Preconditions.checkArgument(Files.exists(path), "File " + rawPath + " does not exist");
     return Optional.of(Paths.get(rawPath));
   }
+
+  private static Optional<URL> maybeUrl(String rawUrl) {
+    if (rawUrl == null) {
+      return Optional.empty();
+    }
+    try {
+      URL url = new URL(rawUrl);
+      return Optional.of(url);
+    } catch (MalformedURLException mURLE) {
```

Comment:
s/mURLE/e

Reply:
I'm not really sure what this means

Reply:
Sorry, I meant that the standard naming convention is "e" rather than "mURLE".

Reply:
ah ok thanks, will do